locust/static/chart.js

```diff
@@ -33,24 +33,25 @@
                     subtext: this.subtitle,
                     x: 10,
                     y: 10,
-                    padding: [-7,0,0,0]
+                    padding: [-7, 0, 0, 0]
                 },
                 tooltip: {
                     trigger: 'axis',
+                    confine: true,
+                    axisPointer: {
+                        animation: true
+                    },
                     formatter: function (params) {
                         if (!!params && params.length > 0 && !!params[0].value) {
                             var str = params[0].name;
-                            for (var i=0; i<params.length; i++) {
+                            for (var i = 0; i < params.length; i++) {
```

Comment:
change `var` to `let`. the difference is the scope of let is smaller than var. so this `i` just visible inside the loop